mobile/router_api_generated.go

```diff
@@ -0,0 +1,237 @@
+// Code generated by falafel 0.3. DO NOT EDIT.
+// source: routerrpc/router.proto
+
+// +build routerrpc
+
+package lndmobile
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+)
+
+func getRouterConn() (*grpc.ClientConn, func(), error) {
+	conn, err := lightningLis.Dial()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	clientConn, err := grpc.Dial("",
+		grpc.WithDialer(func(target string,
+			timeout time.Duration) (net.Conn, error) {
+			return conn, nil
+		}),
+		grpc.WithInsecure(),
+		grpc.WithBackoffMaxDelay(10*time.Second),
+	)
+	if err != nil {
+		conn.Close()
+		return nil, nil, err
+	}
+
+	close := func() {
+		conn.Close()
+	}
+
+	return clientConn, close, nil
+}
+
+// getRouterClient returns a client connection to the server listening
+// on lis.
+func getRouterClient() (routerrpc.RouterClient, func(), error) {
+	clientConn, close, err := getRouterConn()
+	if err != nil {
+		return nil, nil, err
+	}
+	client := routerrpc.NewRouterClient(clientConn)
+	return client, close, nil
+}
+
+// SendPayment attempts to route a payment described by the passed
+// PaymentRequest to the final destination. The call returns a stream of
+// payment status updates.
+//
+// NOTE: This method produces a stream of responses, and the receive stream can
+// be called zero or more times. After EOF error is returned, no more responses
+// will be produced.
+func SendPayment(msg []byte, rStream RecvStream) {
```

Comment:
same name & package as the `SendPayment` method in lightning_api_generated.go.
maybe add a prefix to the methods generated by sub-services?

Reply:
Agree. But as @halseth suggested somewhere else, you can try setting `usePrefex=1` on `gen_bindings.sh`

(Haven't tried though, I manually changed names)

Reply:
Made it possible to add directly to make (https://github.com/lightningnetwork/lnd/pull/3282/commits/73cb325045c3ec874fbf31604254127be0f565b2):
```
make ios tags="routerrpc" prefix=1
```